### PR TITLE
WD_front-end_OTP_Suraj#PR4

### PR DESCRIPTION
**Pull request by SurajKuwar**

### CODE REVIEW

File: next_webapp/public/styles/otp-verification.css
Comment (adrianweerasinghe97) [inline next_webapp/public/styles/otp-verification.css:1]: The CSS looks good

File: next_webapp/src/app/[locale]/otp-verification/page.tsx
Comment (adrianweerasinghe97) [inline next_webapp/src/app/[locale]/otp-verification/page.tsx:1]: The code looks good. No syntax issues to be seen.

**Review by adrianweerasinghe97 — COMMENTED** [review]
1. The CSS looks good
2. You might want to recheck for any imports or any syntax issues with the code

**Comment by kaseyuu** [comment]
Hi Suraj, please revise the CSS file path as indicated
image

**Comment by kaseyuu** [comment]
[Forget password page frontend development (max1)](https://trello.com/c/dxgCwgfg/560-forget-password-page-frontend-development-max1)

**Pull request closed without merging**